mesheryctl/cmd/cleanup.go

```diff
@@ -25,15 +25,19 @@ var cleanupCmd = &cobra.Command{
 	Short: "Cleanup Meshery's configuration",
 	Long:  `Reset Meshery to it's default configuration.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		log.Info("Reseting Meshery config...")
-		if err := downloadFile(dockerComposeFile, fileURL); err != nil {
-			log.Fatal("cleanup cmd: ", err)
-		}
-		log.Info("Meshery config has been reset to its default settings.")
-
+		resetMesheryConfig()
 	},
 }
 
 func init() {
 	rootCmd.AddCommand(cleanupCmd)
 }
+
+// cleans meshery config
```

Comment:
Can we do ~cleans~ reset ?

Reply:
We should then change the whole file name @kushthedude

Reply:
But I read that we are going to deprecate the cleanup command so the things will change then, should this be future thing then?

Reply:
Renaming of file is already taken care in https://github.com/layer5io/meshery/pull/860/files . If #860 gets merged before this PR then we have to make another PR to address this comment.

Reply:
Therefore I believe we should use ~cleanup~ reset wherever we can so that further conflict don't arise with #860

Reply:
So should I just update the comment right?

Reply:
I have updated it @kushthedude